### PR TITLE
feat(kinput): fun with labels

### PR DESCRIPTION
**Pull request by kaiarrowood**
#### Changes made:
* Parameterize `help` text and add a built in `label` prop
* Maintain support for `.help` utility class and combining with `KLabel`
* Maintain attribute binding
* Update snapshots

Help section:

image

Labeling options:

image

image

### PR Checklist
- [x] Does not introduce dependencies
- [x] **Functional:** all changes do not break existing APIs and if so, bump major version.
- [x] **Tests pass:** check the output of yarn test packages/<Kongponent>
- [x] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
- [x] **Framework style:** abides by the essential rules in Vue's style guide
- [x] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs), or leftover comments
- [x] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

**Comment by dustinryerson** [comment]
@kaiarrowood should we add some tests for these changes as well?

**Review by dustinryerson — COMMENTED** [review]
@kaiarrowood should we add some tests for these changes as well?

**Comment by kaiarrowood** [comment]
> @kaiarrowood should we add some tests for these changes as well?

@dustinryerson Definitely, added a couple tests, not sure if the coverage doing it this was is good enough?

File: docs/components/input.md [hunk 1]
```diff
@@ -8,6 +8,53 @@
...
+<KInput help="I can help with that" placeholder="Need help?" />
+
+```
+<KInput help="I can help with that" placeholder="Need help?" />
+```
+
+You also have the option of using the `.help` utility class. This is meant to be used after **KInput** and will be styled appropriately. 
+
+<KInput type="text" placeholder="Need help?" />
+<p class="help">I can help with that</p>
```
Comment (dustinryerson) [inline docs/components/input.md:25]: Non-blocking question: is this supposed to be styled like the input help above?

Screen Shot 2021-07-26 at 2 08 50 PM

**Review by dustinryerson — APPROVED** [review]
Left a non-blocking question, but this looks good, great work!

File: docs/components/input.md (hunk 1 above)
Comment (kaiarrowood) [reply docs/components/input.md:25]: That's really peculiar I'm not seeing this in the netlify build. Are you seeing this locally? If it's local, you have to run the CSS generator command `yarn build:postcss` to pick up the newly generated styles.

image

Comment (dustinryerson) [reply docs/components/input.md:25]: Ah thanks! I also had to run `yarn build:styles`, but that seems to have fixed it. Thanks for the tip!

Comment (kaiarrowood) [reply docs/components/input.md:25]: Ah, sorry! Yeah that's the one to run, `yarn build:styles` calls the postcss script I mentioned above too!

**Pull request merged**